book/validation.rst

```diff
@@ -35,7 +35,7 @@ application. The goal of validation is to tell you if the data
 of an object is valid. For this to work, you'll configure a list of rules
 (called :ref:`constraints <validation-constraints>`) that the object must
 follow in order to be valid. These rules can be specified via a number of
-different formats (YAML, XML, annotations, or PHP).
+different formats (annotations, YAML, XML, or PHP).
```

Comment:
Yea, I thought it wasn't needed but just left it as is, will fix on the next commit.